content/articles/data-preprocessing-python/index.md

```diff
@@ -0,0 +1,248 @@
+---
+layout: engineering-education
+status: publish
+published: true
+url: /data-preprocessing-python/
+title: Getting started with data preprocessing in Python
+description: 
+date: 2021-07-10T00:00:00-18:00
+topics: []
+excerpt_separator: <!--more-->
+images:
+
+  -url: /engineering-education/data-preprocessing-python/hero.jpg
+   alt: Data preprocessing example image
+---
+
+
+Data preprocessing is the first machine learning step in which we transform raw data into a suitable format that makes it easy to implement accurate machine learning models.
+In this article, we will learn all the steps that are involved in the data preprocessing phase. 
+
+### Prerequisites
+
+To follow this tutorial comfortably, we need to meet the following conditions.
+1. Prior knowledge in python programming.
+2. Connected to the internet.
+2. Access to Google Colab [Here](https://colab.research.google.com/notebooks/intro.ipynb).
+
+### Mounting our drive to Google Colab
+
+In this article, we shall carry out our data preprocessing activities on Google Colad. Therefore, we need to ensure our Google Drive is accessible by the Google Colab. To ensure this, first, let's download our data to our computer from [here](https://github.com/Madrinmarison/data).
+
+Since we've successfully downloaded our data, let's now upload it to google drive through [google.drive.com](https://drive.google.com/drive/u/0/my-drive).
+
+Now our data is uploaded to Google Drive.
```

Comment:
Explain the details of the folder in which you want the files saved for a later use.